apps/server/src/infra/etherpad-client/etherpad-client.module.ts

```diff
@@ -0,0 +1,48 @@
+import { DynamicModule, Module } from '@nestjs/common';
+import { AuthorApi, GroupApi, SessionApi } from './etherpad-api-client/api';
+import { Configuration, ConfigurationParameters } from './etherpad-api-client/configuration';
+import { EtherpadClientAdapter } from './etherpad-client.adapter';
+
+export interface EtherpadClientConfig extends ConfigurationParameters {
+	apiKey?: string;
+	basePath?: string;
+	cookieExpiresSeconds: number;
+	cookieReleaseThreshold: number;
+	padURI: string;
+}
+
+@Module({})
+export class EtherpadClientModule {
+	static register(config: EtherpadClientConfig): DynamicModule {
+		const providers = [
+			EtherpadClientAdapter,
+			{
+				provide: GroupApi,
+				useFactory: () => {
+					const configuration = new Configuration(config);
```

Comment:
I'm confused by this because the `Configuration` from the api-client does not know `cookieExpiresSeconds`, `cookieReleaseThreshold` and `padURI`. How does this work? What happens with them? Am I missing something here?